pkg/config/config_internal_test.go

```diff
@@ -1,102 +1,185 @@
 package config
 
 import (
-	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/osv-scanner/pkg/models"
 )
 
-type testStruct struct {
-	targetPath   string
-	config       Config
-	configHasErr bool
+// Attempts to normalize any file paths in the given `output` so that they can
+// be compared reliably regardless of the file path separator being used.
+//
+// Namely, escaped forward slashes are replaced with backslashes.
+func normalizeFilePaths(t *testing.T, output string) string {
+	t.Helper()
+
+	return strings.ReplaceAll(strings.ReplaceAll(output, "\\\\", "/"), "\\", "/")
 }
 
-func TestTryLoadConfig(t *testing.T) {
+func Test_normalizeConfigLoadPath(t *testing.T) {
 	t.Parallel()
 
-	expectedConfig := Config{
-		IgnoredVulns: []IgnoreEntry{
-			{
-				ID: "GO-2022-0968",
+	type args struct {
+		target string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "target does not exist",
+			args: args{
+				target: "./fixtures/testdatainner/does-not-exist",
 			},
-			{
-				ID: "GO-2022-1059",
+			want:    "",
+			wantErr: true,
+		},
+		{
+			name: "target is file in directory with config",
```

Comment:
maybe I miss something - should this be `directory without config`? there isn't `osv-scanner.toml` in `innerFolder`

Reply:
🤦 I had it written without that the first time, but then got confused with the `tryLoadConfig` tests and added it - thanks for catching this!

Reply:
so now there are two `target is file in directory` tests - any difference between two tests?

Reply:
there were always two tests like this - I've just ported them across. I think technically they're different because one has a config file in them and the other doesn't, but the original test didn't make it clear if that was a part of the test or just a coincidence.

I wasn't that fussed because these tests are very fast to run so I didn't have a huge issue with retaining a potentially duplicated test here (given it's not a _total_ duplicate from a file system POV even if that's not super revertant to the implementation)

Reply:
I think it's better to state the difference in the test names - maybe @another-rex knows why we have these tests.

Reply:
fow now I've just adjusted the names to be unique, as I'd like to get this landed ASAP so I can rebase #1248 and #1249